code/TemplateStudioForWinUICs/Templates/_comp/MT/Project.MenuBar/Views/ShellPage.xaml

```diff
@@ -21,27 +22,28 @@
         </SplitView.Pane>
         <Grid>
             <Grid.RowDefinitions>
-                <RowDefinition Height="Auto" />
+                <RowDefinition Height="48" />
                 <RowDefinition Height="*" />
             </Grid.RowDefinitions>
-            <MenuBar VerticalAlignment="Top">
-                <!--
-                TODO: You can edit the text for the menu in String/en-US/Resources.resw
-                Adding <MenuBar.Items> xaml code is a workaround to the following issue
-                https://github.com/microsoft/microsoft-ui-xaml/issues/4385
-                -->
-                <MenuBar.Items>
+            <Grid Grid.Row="0">
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="*" />
+                    <ColumnDefinition Width="Auto" />
```

Comment:
What's this column (definition) for? I can't see why it exists

Reply:
It's for the Settings button when the SettingsPage is added.

Reply:
Thanks. Yes we considered making the composition template more complex but preferred this simpler approach as well.